cinema/views.py

```diff
@@ -1,108 +1,71 @@
+import re
+
 from django.contrib import messages
 from django.contrib.auth.decorators import login_required
-from django.db.models import Q
 from django.shortcuts import get_object_or_404, render
 from django.views.generic import ListView, TemplateView
 
-from .forms import UserCommentToProductForm, GuestCommentToProductForm, \
-    UserCommentToFilmForm, GuestCommentToFilmForm, UserCommentToPersonForm, \
-    GuestCommentToPersonForm, UserCommentToNewsForm, GuestCommentToNewsForm
-from .models import Product, News, Film, CinemaPerson, MpaaRating, \
-    CommentToProduct, CommentToFilm, CommentToPerson, CommentToNews
-
-all_films = Film.objects.prefetch_related(
-    "staff__cinemafilmpersonprofession_set__profession", "staff__user",
-    "country", "genre", "language", "distributor", "news_set"
-).select_related("imdb_rating", "mpaa_rating", "product")
-
-top_films = all_films.values_list(
-    "pk", "title", "imdb_rating__value", "budget", "usa_gross",
-    "world_gross", named=True
+from .forms import (
+    UserCommentToProductForm,
+    GuestCommentToProductForm,
+    UserCommentToFilmForm,
+    GuestCommentToFilmForm,
+    UserCommentToPersonForm,
+    GuestCommentToPersonForm,
+    UserCommentToNewsForm,
+    GuestCommentToNewsForm
```

Comment:
Please always add a trailing comma after the last element in a list.

Reply:
ok